storm-client/src/jvm/org/apache/storm/daemon/worker/WorkerState.java

```diff
@@ -339,6 +344,15 @@ public WorkerState(Map<String, Object> conf, IContext mqContext, String topology
         int maxTaskId = getMaxTaskId(componentToSortedTasks);
         this.workerTransfer = new WorkerTransfer(this, topologyConf, maxTaskId);
         this.bpTracker = new BackPressureTracker(workerId, localTaskIds);
+        // 
```

Comment:
‘//‘ can be removed, and better to extract below to private method as we did above.